CustomerIOSharp/CustomerIO.cs

```diff
@@ -12,10 +12,11 @@
 
     public class CustomerIo
     {
-        private const string Endpoint = "https://track.customer.io/api/v1/";
-
+        private const string TrackEndpoint = "https://track.customer.io/api/v1/";
+        private const string ApiEndpoint = "https://api.customer.io/v1/api/";
```

Comment:
It may be cleaner to refactor `CallMethodAsync` to instead take the entire endpoint (absolute URL) for each method you call. That way `MethodTriggerBroadcast` could be defined as `$"{ApiEndpoint}/campaigns/{campaign_id}/triggers"`, and we could get away with just one parameter to define which method to call.

That would allow dropping settings the base address for the `HttpClient`, which means we could even pass in a pre-existing `HttpClient` to allow for re-use (which is probably a good idea).